File: OOPs_Concepts/src/variables/Student.java
package variables;

public class Student { // POJO or BEAN
	
	 private int sid;
	private String sname;
	private double fee;
	
	
	public Student() {
		
		
		
	}
	
	
	public Student(int sid, String sname, double fee) {
		super();
		this.sid = sid;
		this.sname = sname;
		this.fee = fee;
	}
	public int getSid() {
		return sid;
	}
	public void setSid(int sid) {
		this.sid = sid;
	}
	public String getSname() {
		return sname;
	}
	public void setSname(String sname) {
		this.sname = sname;
	}
	public double getFee() {
		return fee;
	}
	public void setFee(double fee) {
		this.fee = fee;
	}
	
	
	
}

File: ReactJS_Demos/components_props_state/src/Student.js

function  Student(props){

    function add(a,b){

            return a+b;

    }



    return(
        <>
        <h1>Hi , I am  {props.sname} and a student !  {add(4,4)} </h1>
        <h2>Sid: {props.sid}  SName: {props.sname}    Fee: {props.fee}</h2>

        </>
        );


}

export default Student;